src/app/app.module.ts

```diff
@@ -24,7 +24,8 @@ import {ClarityModule} from "../clarity/clarity.module";
         AppComponent,
         LandingComponent
     ],
-    bootstrap: [ AppComponent ]
+    bootstrap: [ AppComponent ],
+    schemas: [ CUSTOM_ELEMENTS_SCHEMA ]
```

Comment:
This is not needed since the module already imports ClarityModule. I'd rather you didn't add the schema, and if you have errors we import ClarityModule from the various modules that need it. That way we can make sure it works without schema for most people. If you need help handling these errors, I'd be happy to.